app/tests/test_serializer.py

```diff
@@ -128,12 +129,191 @@ def test_collection_serialization(self):
         }
         self.check_stac_collection(expected, python_native)
 
-        # back-transate to Python native:
-        stream = io.BytesIO(content)
-        data = JSONParser().parse(stream)
-        # back-translate into fully populated collection instance:
-        serializer = CollectionSerializer(data=data, context={'request': request})
-        self.assertEqual(True, serializer.is_valid(), msg='Serializer data not valid.')
+    def test_collection_deserialization_create_only_required(self):
+        data = OrderedDict([
```

Comment:
according to specs `links` should be in the required minimum, too.

Reply:
Same as above links are generated (self, root, parent, ...)